src/actions/public/simulateDepositETH.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  Transport,
+  PublicClient,
+  Chain,
+  Hex,
+  SimulateContractParameters,
+  SimulateContractReturnType,
+} from 'viem'
+import { l1StandardBridgeABI } from '@eth-optimism/contracts-ts'
+import { OpChainL2 } from '@roninjin10/rollup-chains'
+import { simulateContract } from 'viem/actions'
+
+type DepositETHParameters = {
```

Comment:
can we share this across share and write?

Reply:
Good idea